README.md

```diff
@@ -21,12 +21,30 @@ Create a `.env` file inside the `packages/backend` directory with the following
 ```
 GITHUB_CLIENT_ID=your_github_client_id
 GITHUB_CLIENT_SECRET=your_github_client_secret
+ARCPY_URI=http://127.0.0.1:8045
+ARCPY_ADMIN_USER=someadmin
+ARCPY_ADMIN_PASSWORD=somepassword
+ARCPY_DATABASE=somedatabase
 ```
  
 To generate GitHub OAuth credentials, follow the official Backstage documentation:  
 https://backstage.io/docs/auth/github/provider/
  
-### 3. Start the app
+### 3. Start the Docker containers
+
+```bash
+docker compose -f 'docker-compose.yml' up -d --build 
+```
+
+### 4. Start the arcpy dummy script
+
+In a separate terminal, run:
+
+```bash
+python .\plugins\catalog-backend-module-arcgis-sde-data\src\sde-python-proxy\test.py
+```
```

Comment:
It would make development a lot more straight forward if this was run from docker compose (so that it's started by the previous command).

Something along the lines of (not syntax checked):

```yaml:
services:
  arcsde-proxy:
    image: python:3.12
    command: /app/test.py
    ports:
      - 8045:8045
    volumes:
      - ../plugins/catalog-backend-module-arcgis-sde-data/src/sde-python-proxy/:/app
```

I promise this is the last thing I'll comment on :P